lib/application/constants/strings.dart

```diff
@@ -16,3 +16,30 @@ const collectionsReviewTab = 'Revisar';
 const collectionsSectionHeaderSeeAll = 'Ver todos';
 
 const collectionsMemoryStability = 'Estabilidade da Memória';
+
+const progressTotalStudyTime = 'HORAS TOTAIS DE ESTUDOS';
```

Comment:
Following the same suggestion from the last PR, what about using `toUppercase` in this string use-case?

https://github.com/olmps/memo/pull/44#discussion_r615843902

Reply:
@matuella What about this one? Do you think that it makes sense?